srv/components/misc_pkgs/kafka/sanity_check.sls

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Test kafka:
```

Comment:
This ensure the service is running, doesn't check if it's running.
What @83bhp meant was to use the tests:
- https://github.com/Seagate/cortx-prvsnr/commit/80c70e443d8fc35214d027466ae038a8056a0b23#diff-f2e1b47d101c4aa5f1493a7361ccb300dbbd4e4bfc7d9c510a219cd83a92e59cR22
- https://github.com/Seagate/cortx-prvsnr/commit/80c70e443d8fc35214d027466ae038a8056a0b23#diff-f2e1b47d101c4aa5f1493a7361ccb300dbbd4e4bfc7d9c510a219cd83a92e59cR41

Do update accordingly.

Reply:
This command should be in loop as per my previous comment.